docs/localization.rst

```diff
@@ -67,25 +66,25 @@ Adding a new Locale
 
 #. Create the `jsi18n` file for the new locale::
 
-        $ ./manage.py compilejsi18n
+        make compilejsi18n
 
 #.  Verify django loads new locale without errors by visiting the locale's home
     page. E.g., https://developer-local.allizom.org/ml/
 
 #.  BONUS: Use `podebug` to test a fake translation of the locale::
 
-        $ cd locale
-        $ podebug --rewrite=bracket templates/LC_MESSAGES/django.pot ml/LC_MESSAGES/django.po
-        $ ./compile-mo.sh .
+        cd locale
+        podebug --rewrite=bracket templates/LC_MESSAGES/django.pot ml/LC_MESSAGES/django.po
+        ./compile-mo.sh .
```

Comment:
Dennis can do fake translations, too. It has some of the same transforms that podebug has, but also has some super ones.